docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -0,0 +1,391 @@
+image::images/jenkins-pipeline-build.png[Jenkins pipeline builds]
+
+The Errors overview screen provides a high-level view of the exceptions that CI builds catch.
+Similar errors are grouped to quickly see which ones are affecting your services
+and allow you to take action to rectify them.
+
+[role="screenshot"]
+.Jenkins jobs and pipelines errors in Elastic Observability
+image::images/jenkins-pipeline-errors.png[Jenkins pipeline build errors]
+
+[role="screenshot"]
+.Concourse CI pipeline execution as a trace in Elastic Observability
+image::images/concourse-ci-traces.png[Concourse CI traces view]
+
+[discrete]
+[[ci-cd-developers]]
+== Observability for developers
+
+Development teams need to continuously optimize their ever-changing CI/CD pipelines to improve
+their reliability while chasing for faster pipelines. There are requirements to document CI/CD pipelines
+with trace visualizations to show what’s happening and improve performance and reliability
+(flaky tests and pipelines).
+
+Integrating with many popular CIDevOps tools like Maven or Ansible using OpenTelemetry, Elastic Observability
+solves these problems by providing deep insights into the execution of CI/CD pipelines.
+
+[discrete]
+[[ci-cd-visibility]]
+=== CI/CD pipeline visibility
+
+The visualization of CI/CD pipelines as distributed traces in Elastic Observability provides
+documentation and health indicators of all your pipelines.
+
+The APM Service view in Elastic Observability provides a view of all your instrumented CI/CD
+servers.
+
+[role="screenshot"]
+.Jenkins servers in Elastic Observability
+image::images/jenkins-servers.png[Jenkins servers view]
+
+The Service page provides more granular insights into your CI/CD workflows by breaking down health
+and performance metrics by pipeline. You can sort and filter the list to quickly surface which
+pipelines are the slowest or experience the most errors. In the example below, we have sorted pipelines
+by average build duration to show the slowest ones.
+
+[role="screenshot"]
+.A Jenkins server in Elastic Observability
+image::images/jenkins-server.png[Jenkins server view]
+
+[discrete]
+[[ci-cd-pipelines]]
+=== Individual pipeline visibility
+
+Once you’ve identified the pipeline you want to troubleshoot, you can drill down to get more detailed
+information about its performance over time. The pipeline summary shows a breakdown of duration and
+failure rates across the pipeline’s individual builds and jobs to spot slowdowns or failures.
+
+[role="screenshot"]
+.Performance overview of a Jenkins pipeline in Elastic Observability
+image::images/jenkins-pipeline-overview.png[Jenkins pipeline overview]
+
+The pipelines and traditional jobs are instrumented automatically. If you spot a slow or failing
+build and need to understand what’s happening, you can drill into the trace view of the build to look
+for the high duration or errorful jobs. You can then drill into the details to understand the
```

Comment:
NIT:
Beat with me, I'm not a native speaker and I don't know a synonym for `drill into` , but since there are two occurrences quite closed, if it's possible to use another synonym?